### PR TITLE
Add documentation for objectHeaders

### PR DESCRIPTION
**Pull request by superandrew213**
Fixes https://github.com/MadSkills-io/fullstack-serverless/issues/3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,6 +250,39 @@ The name of your error document inside your `distributionFolder`. This is the fi
 
 ---
 
+**objectHeaders** 
+
+_optional_, no default
+
+```yaml
+custom:
+  client:
```
Comment (andrewphahn) [inline README.md]:
```suggestion
  fullstack:
```

It should be `fullstack` not `client`, right?

**Review by andrewphahn — CHANGES_REQUESTED** [review]
Thank you! This is great. Just one question...

File: README.md (hunk 1 above)
Comment (superandrew213) [reply README.md]: Yes my bad 🤦‍♂️

**Comment by andrewphahn** [comment]
Awesome, thank you!

**Pull request merged**